### PR TITLE
General Polish and Clean Up

### PR DESCRIPTION
**Pull request by b13decker**
Made polish and clean up changes to finalize the Cryptol specs.

### CODE REVIEW

File: spec/Common/ModArith.cry [hunk 1]
```diff
@@ -6,9 +6,9 @@ module Common::ModArith where
 /**
  * Convert from a fixed size bit type to a `Z` type.
  * ```repl
- * :prove (fromIntegral`{13} (3:[384])) == (3 : Z 13)
- * :prove (fromIntegral`{13} (14:[384])) == (1 : Z 13)
- * :prove (fromIntegral`{13} (13:[384])) == (0 : Z 13)
+ * :check (fromIntegral`{13} (3:[384])) == (3 : Z 13)
```
Comment (mccleeary-galois) [inline spec/Common/ModArith.cry:9]: We could use `:exhaust` to show that we are doing a proof by exhaustion.

File: spec/Spec/BlsHelpers.cry [hunk 2]
```diff
@@ -33,6 +40,12 @@ bls_modular_inverse x = toIntegral (modular_inverse x') where
 /**
  * Divide two field elements: x by y
  * @see https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md#div
+ * ```repl
+ * :prove bls_div 1 2 == 0x39f6d3a994cebea4199cec0404d0ec02a9ded2017fff2dff7fffffff80000001
```
Comment (mccleeary-galois) [inline spec/Spec/BlsHelpers.cry:44]: Reason why these are `:prove`?

**Review by mccleeary-galois — APPROVED** [review]
LGTM overall.

File: spec/Spec/BlsHelpers.cry (hunk 2 above)
Comment (b13decker) [reply spec/Spec/BlsHelpers.cry:44]: Looks like I missed these. Thanks for catching it. I'll add a commit.

**Pull request merged**